src/lib/lore.sh

```diff
@@ -98,23 +98,36 @@ function retrieve_available_mailing_lists()
   local flag="$1"
```

Comment:
Revise this function documentation to briefly explain the pagination and why it is done.